pkgs/development/python-modules/keystone-engine/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, stdenv, buildPythonPackage, fetchPypi, keystone }:
+
+buildPythonPackage rec {
+  pname = "keystone-engine";
+  version = "0.9.2";
+
+  src = fetchPypi {
+   inherit pname version;
+   sha256 = "1xahdr6bh3dw5swrc2r8kqa8ljhqlb7k2kxv5mrw5rhcmcnzcyig";
+  };
+
+  preConfigure = if stdenv.isDarwin then ''
```

Comment:
Please do not conditional patch this if it does not break the linux build. This easily bit rots over time and we try to avoid this.

Reply:
Okay, no problem. I rolled it back.